content/en/docs/concepts/configuration/secret.md

```diff
@@ -676,6 +676,35 @@ A container using a Secret as a
 Secret updates.
 {{< /note >}}
 
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
```

Comment:
@wojtek-t , Here are some suggestions.
Could you create a sub-heading to introduce this section?
Right now this content is under `Using Secrets` and `mounted-secrets-are-updated-automatically`.
Not sure if this makes sense?
````
#### Immutable Secrets and ConfigMaps
````

Possible edit to introduce feature:

The Kubernetes alpha feature _Immutable Secrets and ConfigMaps_ provides an option to set individual Secrets and ConfigMaps as immutable. For clusters that extensively use secrets (at least tens of thousands of unique Secret to Pod mounts), preventing changes to this data has the following advantages:
<!-- new line here, then list the advantages -->